source/likely.js

```diff
@@ -4,6 +4,7 @@ var likely = require('./index.js');
 var history = require('./history');
 
 window.addEventListener('load', function () {
+    likely.defaults = likely.getDefaultOptions();
```

Comment:
You mean we can define defaults just right near the https://github.com/ilyabirman/Likely/blob/master/source/index.js#L23 , where it's used?